wfe/web-front-end.go

```diff
@@ -42,6 +42,8 @@ const (
 
 	// StatusRateLimited is not in net/http
 	StatusRateLimited = 429
+	// statusBadNonce is used to force sendError to send the proper error type
+	statusBadNonce = 0400
```

Comment:
Yeah so `0400` is used here just as a sentinel to be passed out of `statusCodeFromError` because `sendError` uses it to determine the problem document type and `400` is already used for `core.MalformedProblem`.

Making it decimal definitely makes sense (not really sure why I used octal to begin with).